### PR TITLE
Show how to achieve several menus with whichkey

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: docs/whichkey/usage.md [hunk 1]
```diff
@@ -193,6 +193,25 @@ An example of a `settings.json` file that overrides the space menu is as follows
 The default value can be found in the `contributes.configuration.whichkey.bindings.default` section of the `package.json` in this repo.
 You can use the default value as an example to craft your own custom menu.
 
+### Multiple Menus
+
+Only a single global menu is supported natively.
```
Comment (stevenguh) [inline docs/whichkey/usage.md]: That's not entirely correct, see https://vspacecode.github.io/docs/whichkey/reference#show. One can pass an array of the bindings directly.

File: docs/whichkey/usage.md [hunk 2]
```diff
@@ -193,6 +193,25 @@ An example of a `settings.json` file that overrides the space menu is as follows
 The default value can be found in the `contributes.configuration.whichkey.bindings.default` section of the `package.json` in this repo.
 You can use the default value as an example to craft your own custom menu.
 
+### Multiple Menus
```
Comment (stevenguh) [inline docs/whichkey/usage.md]:
```suggestion
### Bind shortcuts to submenu
```

File: docs/whichkey/usage.md [hunk 3]
```diff
@@ -193,6 +193,25 @@ An example of a `settings.json` file that overrides the space menu is as follows
...
+
+Only a single global menu is supported natively. 
+However, it is possible to simulate several menus by executing a `whichkey.triggerKey` command right after `whichkey.show` to select a submenu. 
+Here's an example submenu keybinding using [ryuta46.multi-command](https://marketplace.visualstudio.com/items?itemName=ryuta46.multi-command) extension:
```
Comment (stevenguh) [inline docs/whichkey/usage.md]:
```suggestion
To open a submenu with a shortcut, one can execute a `whichkey.triggerKey` command right after `whichkey.show` to select a submenu. 
Here's an example `keybindings.json` using [ryuta46.multi-command](https://marketplace.visualstudio.com/items?itemName=ryuta46.multi-command):
```

**Review by stevenguh — COMMENTED** [review]
Thank you for contributing to our docs :) I think this might work better if this is in the `extra.md` page.

**Comment by matklad** [comment]
moved to extra!

**Review by stevenguh — APPROVED** [review]

**Comment by marcoieni** [comment]
Thanks Alex, both for this pr and rust analyzer, which I use every day :)

**Pull request merged**